cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/weights/pallet_xcm.rs

```diff
@@ -373,4 +373,8 @@ impl<T: frame_system::Config> pallet_xcm::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().reads(1))
 			.saturating_add(T::DbWeight::get().writes(1))
 	}
+
+	fn set_asset_claimer() -> Weight {
+		Weight::from_parts(2_176_000, 0)
```

Comment:
was this manually set? weight here is 3x larger than on the other runtimes, also there's no autogenerated comments like:
```
		// Proof Size summary in bytes:
		//  Measured:  `0`
		//  Estimated: `0`
		// Minimum execution time: 707_000 picoseconds.
```

Reply:
Yes. 
I don't remember the exact reason but TTBOMK it was needed for the bot to be able to populate all the other benchmarks.

Reply:
Either way the compiler was complaining on this function and I had to remove it.